mpisppy/phbase.py

```diff
@@ -810,17 +811,10 @@ def _vb(msg):
 
         self._update_E1()  # Apologies for doing this after the solves...
         if (abs(1 - self.E1) > self.E1_tolerance):
-            if self.cylinder_rank == 0:
-                print("ERROR")
-                print("Total probability of scenarios was ", self.E1)
-                print("E1_tolerance = ", self.E1_tolerance)
-            quit()
+            raise RuntimeError(f"Total probability of scenarios was {self.E1};  E1_tolerance = ", self.E1_tolerance)
```

Comment:
Don't you want to keep the cylinder rank 0 check? To avoid every rank throwing the exception? The runtime error is definitely better than the quit().